tmt/base.py

```diff
@@ -656,6 +660,8 @@ def _match(
     @staticmethod
     def create(name, template, tree, force=False):
         """ Create a new story """
+        dry = True if Story._opt('dry') else False
```

Comment:
`Story._opt('dry')` already returns a `bool` so there's no need to check/convert it here.

```suggestion
        dry = Story._opt('dry')
```